### PR TITLE
Support --gpu flag

### PR DESCRIPTION
**Pull request by moshe-kabala**

### CODE REVIEW

File: cmd/local/install.go [hunk 1]
```diff
@@ -134,8 +137,12 @@ func getLatestImages() ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	k3dVersion := k3d.K3sVersion
```
Comment (arikmaor) [inline cmd/local/install.go:140]: it's `k3s` version, not `k3d`

Comment (moshe-kabala) [reply cmd/local/install.go:141]: Right fixed

File: cmd/local/install.go [hunk 2]
```diff
@@ -81,7 +84,8 @@ func init() {
 	}
 
 	cmd.Flags().UintVarP(&port, "port", "p", 4589, "Port to be used for tensorleap installation")
-	cmd.Flags().UintVar(&registryPort, "registry-port", 5699, "Port to be used for docker registyr")
+	cmd.Flags().UintVar(&registryPort, "registry-port", 5699, "Port to be used for docker registry")
+	cmd.Flags().BoolVarP(&useGpu, "gpu", "g", false, "Is use GPU")
```
Comment (arikmaor) [inline cmd/local/install.go:88]:
```suggestion
	cmd.Flags().BoolVar(&useGpu, "gpu", false, "Enable GPU usage for training and evaluating")
```

**Review by arikmaor — APPROVED** [review]
LGTM

**Pull request merged**